command_line/resolutionizer.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import absolute_import, division, print_function
 
```

Comment:
General comment: I remain to be convinced that `dials.resolutionizer` is a _good_ name 🙂

Reply:
`dials.estimate_resolution` > `dials.resolution` > `dials.resolutionizer` IMHO

Reply:
`dials.estimate_resolution` is verbose but correct, and passes the Ronseal test...

Reply:
Resolved in c004a30